spec/features/user_adds_outcome_coverage_to_a_course_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+feature "user adds outcome coverage to a course" do
+  scenario "successfully" do
+    subject = create(:subject)
+    course = create(:course)
+    outcome = create(:outcome, course: course)
+    user = user_with_assessments_access_to(course.department)
+
+    visit manage_assessments_course_outcome_coverages_path(course, as: user)
+    click_on I18n.t('manage_assessments.outcome_coverages.index.add_a_class')
+    select subject.title, from: "outcome_coverage[subject_id]"
+    select outcome.nickname, from: "outcome_coverage[outcome_id]"
+    click_button I18n.t('helpers.submit.outcome_coverage.create')
```

Comment:
I18n is included in tests, so I think you can just do `t(...)` here.

https://github.com/MIT-IR/abet/blob/master/spec/support/I18n.rb